mlflow/spark.py

```diff
@@ -558,6 +586,9 @@ def _validate_model(spark_model):
     from pyspark.ml import Transformer as PySparkTransformer
     from pyspark.ml.util import MLReadable, MLWritable
 
+    if _is_spark_connect_model(spark_model):
```

Comment:
Is spark model always writable?

Reply:
According to design, all models should support read/write